src/components/policy/test/generated_code_test.cc

```diff
@@ -48,21 +49,29 @@ namespace policy {
 TEST(PolicyGeneratedCodeTest, TestValidPTPreloadJsonIsValid) {
   std::ifstream json_file("sdl_preloaded_pt.json");
   ASSERT_TRUE(json_file.is_open());
-  Json::Value valid_table;
-  Json::Reader reader;
-  ASSERT_TRUE(reader.parse(json_file, valid_table));
-  Table table(&valid_table);
+  std::stringstream buffer;
+  buffer << json_file.rdbuf();
+  std::string str(buffer.str());
+  utils::json::JsonValue::ParseResult result =
+      utils::json::JsonValue::Parse(str);
+  utils::json::JsonValue valid_table = result.first;
+  utils::json::JsonValueRef json_value_ref = valid_table;
+  Table table(json_value_ref);
```

Comment:
@OHerasym In bot tests you use absolutely the same code. Please move it out to separate function and use just function call.